addon/components/hyper-table-v2/search.hbs

```diff
@@ -1,15 +1,6 @@
-<OSS::InputContainer
-  @value={{this.searchQuery}}
+<OSS::SearchField
+  data-control-name="table_search_input"
```

Comment:
Nit: We usually put `data-control-name` attributes at the bottom of the attributes in a tag